zgw/portaaltaak/src/main/kotlin/com/ritense/portaaltaak/PortaaltaakPlugin.kt

```diff
@@ -47,6 +47,9 @@ import com.ritense.zakenapi.domain.rol.RolNietNatuurlijkPersoon
 import com.ritense.zakenapi.domain.rol.RolType
 import com.ritense.zakenapi.link.ZaakInstanceLinkNotFoundException
 import com.ritense.zakenapi.link.ZaakInstanceLinkService
+import mu.KLogger
+import mu.KotlinLogging
+import mu.withLoggingContext
```

Comment:
Could you include 
`implementation project(":logging")`
And use this import instead?
```suggestion
import com.ritense.logging.withLoggingContext
```

This new `withLoggingContext` is the same but will also save the logging-context for exceptions